lib/weekly_confirmations_report.rb

```diff
@@ -61,9 +60,11 @@ def refresh
 
   def csv
     CSV.generate(headers: true) do |csv|
-      csv << ['Prison'] + week_range.map { |weekno| @start_of_year + weekno * 7 } + ['NOMIS ID']
+      csv << ['Prison'] + week_range.
+        map { |weekno| @start_of_year + weekno * 7 } + ['NOMIS ID']
       @dataset.keys.sort.each do |prison|
-        csv << [@prison_labeling_function.call(prison)] + week_range.map { |weekno| @dataset[prison][weekno] } + [prison]
+        csv << [@prison_labeling_function.call(prison)] + week_range.
+          map { |weekno| @dataset[prison][weekno] } + [prison]
```

Comment:
I think it's better to split on `+` than `.` here: it breaks up higher-level components of the line.